apis/stargate/v1alpha1/stargate_types.go

```diff
@@ -43,7 +43,7 @@ type StargateTemplate struct {
 	// ContainerImage is the image characteristics to use for Stargate containers. Leave nil
 	// to use a default image.
 	// +optional
-	// +kubebuilder:default={repository:"stargateio", tag:"v1.0.45"}
+	// +kubebuilder:default={repository:"stargateio", tag:"v1.0.66"}
```

Comment:
`-Dstargate.cql.config_path` was introduced in Stargate 1.0.64. I went with the latest.